csm_test_utils/autoscaling/loadbalancer.py

```diff
@@ -9,24 +9,31 @@
 from ..common import Client, base_parser, sub_parsers
 
 AS_LOADBALANCER = "as_loadbalancer"
+CSM_EXCEPTION = "csm_exception"
 LOGGER = logging.getLogger(__name__)
 LOGGER.setLevel(logging.DEBUG)
 
 
 def report(client: Client):
     """Send request and write metrics to telegraf"""
-    target_req = requests.get(client.url, headers={"Connection": "close"})
-    influx_row = Metric(AS_LOADBALANCER)
-    if target_req.status_code == 200:
-        influx_row.add_tag("state", "connected")
-        influx_row.add_tag("host", "scn4")
-        influx_row.add_tag("reason", "ok")
-        influx_row.add_value("elapsed", target_req.elapsed.microseconds / 1000)
-    else:
-        influx_row.add_tag("state", "connection_lost")
-        influx_row.add_tag("host", "scn4")
-        influx_row.add_tag("reason", "fail")
-        influx_row.add_value("elapsed", target_req.elapsed.microseconds / 1000)
+    try:
+        target_req = requests.get(client.url, headers={"Connection": "close"})
+        influx_row = Metric(AS_LOADBALANCER)
+        if target_req.status_code == 200:
+            influx_row.add_tag("state", "connected")
+            influx_row.add_tag("host", "scn4")
+            influx_row.add_tag("reason", "ok")
+            influx_row.add_value("elapsed", target_req.elapsed.microseconds / 1000)
+        else:
+            influx_row.add_tag("state", "connection_lost")
+            influx_row.add_tag("host", "scn4")
+            influx_row.add_tag("reason", "fail")
+            influx_row.add_value("elapsed", target_req.elapsed.microseconds / 1000)
+    except Exception as E:
```

Comment:
Maybe except more narrow `IOError` and log them as `as_lb_unavailable`?

Reply:
i think it is good to use one "csm_exception" metric for all exceptions that we can obtain in future from any kinds of test utils, and we can differ it by tag "Reporter"(e.g.: as_loadbalancer). And do not create any other metrics for exceptions.